tests/regression/router-sanitizer.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+source tests/regression/common.sh
+
+check_env
+setup_sanitizer_opts
+
+# Get command line arguments
+# Current arguments parsed are <sleep_time> <remaining_args>
+get_cli_args $@
+
+# Install prerequisites
+make
+pip install --user -r tests/regression/requirements.txt
+redis-server &
```

Comment:
Check the status of Redis server.

Reply:
I will open another issue for this requirement

Reply:
Add `FIXME` for future improvements.